src/features/unified-share-modal/UnifiedShareModal.js

```diff
@@ -45,7 +45,7 @@ class UnifiedShareModal extends React.Component<USMProps, State> {
             getInitialDataCalled: false,
             isConfirmModalOpen: false,
             isEmailLinkSectionExpanded: false,
-            isFetching: false,
```

Comment:
Why do we change this default?

Reply:
It was originally set to `true`. I accidentally changed it to `false` in my last PR: https://github.com/box/box-ui-elements/pull/2129/files#diff-a6be9fd7eb644ea8a0a16cc96b5a65b5L179. This ensures that the loading indicator shows up on load.